bridge/build.gradle.kts

```diff
@@ -0,0 +1,51 @@
+import org.jetbrains.kotlin.gradle.dsl.JvmTarget
+
+// For more info on versioning, see the README.
+val version = "0.1.0"
+
+plugins {
+    alias(libs.plugins.android.library)
+    alias(libs.plugins.kotlin.android)
+    alias(libs.plugins.kotlin.parcelize)
+}
+
+android {
+    namespace = "com.bitwarden.bridge"
+    compileSdk = 34
+
+    defaultConfig {
+        minSdk = 28
+
+        testInstrumentationRunner = "androidx.test.runner.AndroidJUnitRunner"
+        consumerProguardFiles("consumer-rules.pro")
+        buildConfigField("String", "VERSION", "\"$version\"")
+    }
+
+    buildTypes {
+        release {
+            isMinifyEnabled = false
```

Comment:
This should be on, right?

Reply:
Because this is an SDK, we don't want to minify prematurely.

Reply:
Does that mean it will be turned on later, or that it not be turned on and will rely on the consumer to perform minification and declare the library proguard rules?

Reply:
The idea is that this will remain off and if the consumer wants to minify later they can. And as we add code here, we will add the library Proguard rules to ensure minification does not break the library itself.